INSTALLATION.md

```
@@ -0,0 +1,337 @@
+
+```bash
+npm install -g typescript
+```
+
+This command installs TypeScript globally on your system so that it can be accessed from any project.
+
+## Install Required Packages
+
+Run the following command to install the packages and dependencies required by the app:
+
+```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)